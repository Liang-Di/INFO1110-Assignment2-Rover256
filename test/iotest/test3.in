START f
STATS
SCAN shade
SCAN elevation
MOVE W 1
STATS
WAIT 1
STATS
FINISH
QUIT
