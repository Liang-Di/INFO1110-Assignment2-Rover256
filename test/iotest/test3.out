
Explored: 3%
Battery: 100/100


| | |#| | |
| |#|#|#| |
| | |H| | |
|#| | |#| |
| | | |#| |


|-|-|-|-|-|
|-|-|-|-|-|
|-| |H|-|-|
|-|+|-|-|-|
|-|-|+|+|-|


Explored: 83%
Battery: 99/100


Explored: 83%
Battery: 100/100


You explored 83% of pluto

